nb/api/align.md

```diff
@@ -25,40 +25,40 @@ Box(10, 10, 1).view().note('Box(10, 10, 1)');
 
 ![Image](align.md.$3.png)
 
-by(align())
+align()
 
 ```JavaScript
-Box(10, 10, 1).by(align()).and(origin).view().md('by(align())');
+Box(10, 10, 1).align().and(origin).view().md('align()');
 ```
 
 ![Image](align.md.$4.png)
 
-by(align('x>'))
+align('x>')
 
 ```JavaScript
-Box(10, 10, 1).by(align('x>')).and(origin).view().md("by(align('x>'))");
+Box(10, 10, 1).align('x>').and(origin).view().md("align('x>')");
 ```
 
 ![Image](align.md.$5.png)
 
 align('x<')
 
 ```JavaScript
-Box(10, 10, 1).by(align('x<')).and(origin).view().md("align('x<')");
+Box(10, 10, 1).align('x<').and(origin).view().md("align('x<')");
```

Comment:
This is way easier to read I think

Reply:
Yes, I think so -- using alignment() should be an unusual case.